vercel.json

```diff
@@ -1991,11 +1991,11 @@
       "destination": "/encyclopedia/go-sdk-multithreading"
     },
     {
-      "source": "/dev-guide/python/async-vs-sync",
+      "source": "/develop/python/async-vs-sync",
       "destination": "/encyclopedia/python-sdk-sync-vs-async"
     },
     {
-      "source": "/dev-guide/python/sandbox",
+      "source": "/develop/python/sandbox",
       "destination": "/encyclopedia/python-sdk-sandbox"
```

Comment:
These 2 we won't want to change, because the slug was "dev-guide" when we moved them.